ios/SmartVillageApp.xcodeproj/project.pbxproj

```diff
@@ -19,7 +19,7 @@
 
 /* Begin PBXFileReference section */
 		008F07F21AC5B25A0029DE68 /* main.jsbundle */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = main.jsbundle; sourceTree = "<group>"; };
-		13B07F961A680F5B00A75B9A /* SmartVillageApp.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = SmartVillageApp.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		13B07F961A680F5B00A75B9A /* app.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = app.app; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
why did this rename to "app.app" instead of the real name "SmartVillageApp.app"? there would be less changes if we would revert this naming.

Reply:
It made this change automatically. 🤷‍♂️

Reply:
everytime i build on my system SmartVillageApp.app is the outcome. please check again on your system in this branch when you run the setup again, what the outcome is. we could rebase then if its the same for you as for me.

Reply:
I solved this problem, it was automatically updated as `app.app` because I did `prebuild` with my `expo` account. When I prebuild with `ikusei`s `expo` account, it changed to `SmartVillageApp.app`.

Reply:
good to know